apl-core/src/main/java/com/apollocurrency/aplwallet/apl/exchange/service/DexService.java

```diff
@@ -92,4 +94,22 @@ public boolean widthraw(String account,String secretPhrase,BigDecimal amount,Str
     }
 
 
+    public void closeOverdueOrders(){
+        List<DexOffer> offers = dexOfferDao.getOverdueOrders(epochTime.getEpochTime());
+
+        for (DexOffer offer : offers) {
+            try {
+                Account account = Account.getAccount(offer.getAccountId());
+                offer.setStatus(OfferStatus.CLOSED);
+
+                dexOfferTable.insert(offer);
+                account.addToUnconfirmedBalanceATM(LedgerEvent.TRANSACTION_FEE, offer.getTransactionId(), offer.getOfferAmount());
+            } catch (Exception ex){
+                LOG.error(ex.getMessage(), ex);
```

Comment:
Better throw exception

Reply:
It will mean that one broken order stop processing.

Reply:
ok